mono/mini/driver.c

```diff
@@ -696,10 +728,12 @@ interp_regression (MonoImage *image, int verbose, int *total_run)
 	guint32 i;
 	int total;
 
+	printf (">1.0\n");
```

Comment:
Debug spew. Comment or remove.

Reply:
Let me fix that by [WIP/DONT MERGE] this PR